### DISCUSSION TITLE
New floppy format "Gem S2/S3, WX, WK4-8, Genesys"

### DISCUSSION THREAD

**Discussion by flyman02** [keirf/greaseweazle#322] [General]
HI,
I wanted to make a contribution to the fantastic Greasewezle project add a new floppy format.
It is a 1.62MB Custom format which is used in the Music Instruments Generalmusic S2/S3, WX, WK4-8, Genesys.
I did some tests and it works well by exporting and importing into .img with these parameters:

disk gemS_wx_wk.1600
    cyls = 80
    heads = 2
    tracks * ibm.mfm
        secs = 10
        bps = 1024
        gap3 = 116
        id = 1
        rate = 500
        cskew = 5
    end
end

I hope to be of help to someone.

**Comment by keirf** [comment]
Thanks. This exists as `akai.1600` but I will include this as an alias `gem.1600`
